tests/test_all.py

```diff
@@ -139,7 +140,9 @@ def test_glibc_present(auto_container):
 def test_zypper_dup_works(container_per_test: ContainerData) -> None:
     """Check that there are no packages installed that we wouldn't find in SLE
     BCI repo by running :command:`zypper -n dup` and checking that there are no
-    downgrades or arch changes.
+    downgrades or arch changes. Then validate that SLE_BCI provides all the packages
```

Comment:
Isn't the `--allow-downgrade` argument in the `zypper` call in line 156 contradicting the "checking that there are no downgrades" statement? :thinking:

Reply:
yes, fixed.